workflows/chipseq/Snakefile

```diff
@@ -624,6 +624,48 @@ rule macs2:
     wrapper:
         wrapper_for('macs2/callpeak')
 
+# Epic2 peak caller
+# See https://github.com/biocore-ntnu/epic2
+rule epic2:
+    """
+    Run the epic2 peak caller
+    """
+    input:
+        ip=lambda wc:
+            expand(
+                c.patterns['merged_techreps'],
+                label=chipseq.samples_for_run(config, wc.epic2_run, 'epic2', 'ip'),
+            ),
+        control=lambda wc:
+            expand(
+                c.patterns['merged_techreps'],
+                label=chipseq.samples_for_run(config, wc.epic2_run, 'epic2', 'control'),
+            )
+    output:
+        bed=c.patterns['peaks']['epic2']
+    resources:
+        mem_mb=gb(16),
+        runtime=autobump(hours=2)
+    log:
+        c.patterns['peaks']['epic2'] + '.log'
+    params:
+        block=lambda wc: chipseq.block_for_run(config, wc.epic2_run, 'epic2'),
+        genome=config['genome']
+    run:
+        # `-c` has to be skipped if no control is provided
+        if len(input.control) > 0:
+            cmd = '-c {input.control} '
```

Comment:
Since `cmd` is at most 2 args rather than the whole command, I think it would make more sense to change `cmd` to `arguments`